change/@microsoft-teams-js-3d42bacc-ebe5-46c9-82ff-61a56c6863e5.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Added new liveShare hub",
```

Comment:
Can you please add a little more detail to this (and if referencing a namespace or type in the code, please surround with backtick quotes, i.e., '\`')? Also "hub" and "host" tend to have a distinct meaning that I don't think this aligns with, so would prefer to use different wording.

Something like "Added \`liveShare\` capability to interact with @microsoft/live-share package, enabling the Live Share feature" or something?

Reply:
Updated